sections/shared/Layout/UserMenu/UserMenu.tsx

```diff
@@ -219,8 +227,16 @@ const Menu = styled.div`
 	grid-auto-flow: column;
 `;
 
-const EnsName = styled.span`
+const Ens = styled.span`
+	display: flex;
+	align-items: center;
 	text-transform: lowercase;
+	gap: 5px;
```

Comment:
unfortunately gap is not supported in safari when using flexbox. 
There are not many users that connect via safari to our dapp but still, would be annoying for them. 
Can you make the spacing with margin? Or something else please ?

Reply:
sure